src/frontend/src/optimizer/mod.rs

```diff
@@ -332,9 +319,6 @@ impl PlanRoot {
         // Logical optimization
         let mut plan = self.gen_optimized_logical_plan();
 
-        // Batch specific logical optimization
-        plan = self.batch_logical_optimize(plan);
-
```

Comment:
Why this is no longer need?

Reply:
Index selection has been moved to `to_batch()` of `LogicalScan`.